css-two-column-layout/styles.css

```diff
@@ -0,0 +1,84 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding: 65px;
+}
+
+.container {
+  font-family: 'Karla', sans-serif;
+  width: 1285px;
+  margin: auto;
+}
+
+.header {
+  text-align: center;
+  width: 50%;
+  margin: auto;
+}
+
+h1 {
+  font-size: 50px;
+  font-weight: 550;
+  text-transform: uppercase;
+  letter-spacing: 5px;
+  line-height: 35px;
+}
+
+p.header-paragraph {
+  color: #616161;
+  font-weight: 300;
+  line-height: 30px;
+}
+
+.row {
+  display: flex;
+}
+
+.top-left,
+.top-right,
+.bottom-left,
+.bottom-right {
```

Comment:
Instead of this, I would suggest using a `column-half` class with `width: 50%` in order to have a proper layout class system